lego/apps/events/models.py

```diff
@@ -605,6 +605,11 @@ def is_admitted(self, user):
             return self.user_reg[0].pool is not None
         return self.registrations.filter(user=user).exclude(pool=None).exists()
 
+    def is_on_waiting_list(self, user):
+        return self.registrations.filter(
+            user=user, pool=None, status=constants.SUCCESS_REGISTER
+        ).exists()
```

Comment:
wait, how does this check if someone's on waiting list? Is the waiting list pool `None`?

Reply:
yes. If a registration is on the waiting list, it's pool is `None`. Of course it has to be a `success` as well :)